guacamole/src/main/frontend/src/app/rest/services/connectionService.js

```diff
@@ -24,9 +24,11 @@ angular.module('rest').factory('connectionService', ['$injector',
         function connectionService($injector) {
 
     // Required services
-    var requestService        = $injector.get('requestService');
     var authenticationService = $injector.get('authenticationService');
     var cacheService          = $injector.get('cacheService');
+
+    // Required types
+    const Error = $injector.get('Error');
```

Comment:
Is this used? I don't see any usages in the relevant part of the diff.

Reply:
Nope. I'll get rid of it.